versions/kruise/next/values.yaml

```diff
@@ -7,6 +7,9 @@ crds:
 installation:
   namespace: kruise-system
   createNamespace: true
+  daemonSet:
+    namespace: kruise-daemon-config
```

Comment:
we should rename namespace to config-namespace so as to avoid the confusion between installation namespace

Reply:
i agree but this will probably turn it into a major release, right?

Reply:
it used to save the pull secrets used by imagepulljobs, so it is not the same as the daemon installation namespace